gpflow/base.py

```diff
@@ -105,6 +116,24 @@ class PriorOn(Enum):
 
 
 class Parameter(tfp.util.TransformedVariable):
+    """A parameter retains both constrained and unconstrained representations. If no transform
+    is provided, these two values will be the same.  It is often challenging for humans to operate with
+    unconstrained parameters, although this is typically easier for the optimiser. For example, a variance cannot be negative, therefore we need a
```

Comment:
Isn't this giving "line too long" linting errors? Either way, should probably rejustify.